seesaw/dataset_tools.py

```diff
@@ -57,5 +58,10 @@ def __len__(self):
 
     def __getitem__(self, idx):
         relpath = self.paths[idx].lstrip("./")
-        image = PIL.Image.open("{}/{}".format(self.root, relpath))
-        return {"file_path": relpath, "dbidx": idx, "image": image}
+        try:
+            image = PIL.Image.open("{}/{}".format(self.root, relpath))
+            return {"file_path": relpath, "dbidx": idx, "image": image}
+
+        except:
+            return "{}/{}".format(self.root, relpath)
```

Comment:
let's try to stick to returning a dictionary, perhaps with a null image to indicate error.